students/olszewski_bartosz/lesson_06_dictionaries/acces_rights.py

```diff
@@ -0,0 +1,27 @@
+number_of_files = int(input())
+file_access_rights = dict()
+
+for file in range(number_of_files):
+    entry = input().split(' ')
+    file_access_rights[entry[0]] = entry[1:]
+
+number_of_operations = int(input())
+for operation in range(number_of_operations):
+    access_denied = True
+    operation_entry = input().split(' ')
+    if operation_entry[0] == "read":
```

Comment:
Please try to avoid magic numbers like 0,1

FILE_ACTION = 0
if operation_entry[FILE_ACTION ] == "read":